setup.py

```diff
@@ -57,7 +57,75 @@ def do_setup():
+                'local_oscilator_drift.py = mintpy.local_oscilator_drift:main',
+                'lookup_geo2radar.py = mintpy.lookup_geo2radar:main',
+                'mask.py = mintpy.mask:main',
+                'modify_network.py = mintpy.modify_network:main',
+                'multilook.py = mintpy.multilook:main',
+                'multi_transect.py = mintpy.multi_transect:main',
+                'plot_coherence_matrix.py = mintpy.plot_coherence_matrix:main',
+                'plot_network.py = mintpy.plot_network:main',
+                'plot_transection.py = mintpy.plot_transection:main',
+                'prep_aria.py = mintpy.prep_aria:main',
+                'prep_cosicorr.py = mintpy.prep_cosicorr:main',
+                'prep_fringe.py = mintpy.prep_fringe:main',
+                'prep_gamma.py = mintpy.prep_gamma:main',
+                'prep_giant.py = mintpy.prep_giant:main',
+                'prep_gmtsar.py = mintpy.prep_gmtsar:main',
+                'prep_hyp3.py = mintpy.prep_hyp3:main',
+                'prep_isce.py = mintpy.prep_isce:main',
+                'prep_roipac.py = mintpy.prep_roipac:main',
+                'prep_snap.py = mintpy.prep_snap:main',
+                'reference_date.py = mintpy.reference_date:main',
+                'reference_point.py = mintpy.reference_point:main',
+                'remove_hdf5_dataset.py = mintpy.remove_hdf5_dataset:main',
+                'remove_ramp.py = mintpy.remove_ramp:main',
+                'save_gbis.py = mintpy.save_gbis:main',
+                'save_gdal.py = mintpy.save_gdal:main',
+                'save_gmt.py = mintpy.save_gmt:main',
+                'save_hdfeos5.py = mintpy.save_hdfeos5:main',
+                'save_kite.py = mintpy.save_kite:main',
+                'save_kmz.py = mintpy.save_kmz:main',
+                'save_kmz_timeseries.py = mintpy.save_kmz_timeseries:main',
+                'save_qgis.py = mintpy.save_qgis:main',
+                'save_roipac.py = mintpy.save_roipac:main',
+                'smallbaselineApp.py = mintpy.smallbaselineApp:main',
+                'solid_earth_tides.py = mintpy.solid_earth_tides:main',
+                'spatial_average.py = mintpy.spatial_average:main',
+                'spatial_filter.py = mintpy.spatial_filter:main',
+                'subset.py = mintpy.subset:main',
+                'temporal_average.py = mintpy.temporal_average:main',
+                'temporal_derivative.py = mintpy.temporal_derivative:main',
+                'temporal_filter.py = mintpy.temporal_filter:main',
+                'timeseries2velocity.py = mintpy.timeseries2velocity:main',
+                'timeseries_rms.py = mintpy.timeseries_rms:main',
+                'tropo_gacos.py = mintpy.tropo_gacos:main',
+                'tropo_phase_elevation.py = mintpy.tropo_phase_elevation:main',
+                'tropo_pyaps3.py = mintpy.tropo_pyaps3:main',
+                # 'tropo_pyaps.py = mintpy.tropo_pyaps:main',
```

Comment:
Let's keep both as `tropo_pyaps.py` is not completely replaced by `tropo_pyaps3.py` yet.

Reply:
`tropo_pyaps.py` will fail with a:
```sh
$ tropo_pyaps.py
Traceback (most recent call last):
  File "/home/jhkennedy/Documents/Code/contributing/MintPy/mintpy/tropo_pyaps.py", line 14, in <module>
    import pyaps as pa
ModuleNotFoundError: No module named 'pyaps'
```
due to [this import](https://github.com/jhkennedy/MintPy/blob/entrypints/mintpy/tropo_pyaps.py#L13-L16) of the older (Python 2?) `pyaps` package` which isn't listed as a dependency or install in the environments.

Could it be converted to `import pyaps3 as pa`? Otherwise, I don't think it can/should be listed as an entrypoint.

Reply:
`pyaps` package was distributed under Caltech/Earthdef. The website went down recently. I suppose there are still users (including me occasionally) using that via `tropo_pyaps.py`.

Reply:
Alright, I'll include it, but it'd probably be worth improving the [import error message](https://github.com/jhkennedy/MintPy/blob/entrypints/mintpy/tropo_pyaps.py#L16) to explain what pyaps is and where to get it.